docs/api/README.md

```diff
@@ -895,6 +896,36 @@ const debounce = (ms, pattern, task, ...args) => fork(function*() {
 ### `debounce(ms, channel, saga, ...args)`
 You can also handel a channel as argument and the behaviour is the same as [`debounce(ms, pattern, saga, ..args)`](#debouncems-pattern-saga-args)
 
+### `retry(maxTries, delay, fn, ...args)`
+
+Creates an Effect description that instructs the middleware to call the function `fn` with `args` as arguments while `fn` does not return the result or `fn` calls count less then `maxTries`. `Delay` instructs `retry` to make a pause between `fn` calls.
```

Comment:
result or --> result of
then --> than

Reply:
probably nitpick - from the first look is not clear that effect try to call `fn` again if the previous call threw an error